src/PluginClient.php

```diff
@@ -62,6 +62,36 @@ public function __construct($client, array $plugins = [], array $options = [])
     }
 
     /**
+     * Adds a plugin to the client.
+     *
+     * @param Plugin $plugin
+     */
+    public function addPlugin(Plugin $plugin)
+    {
+        $this->plugins[] = $plugin;
+    }
+
+    /**
+     * Get all active plugins.
+     *
+     * @return Plugin[]
+     */
+    public function getPlugins()
+    {
+        return $this->plugins;
+    }
+
+    /**
+     * Assign plugins to the client.
+     *
+     * @param Plugin[] $plugins
+     */
+    public function setPlugins(array $plugins = [])
+    {
+        $this->plugins = $plugins;
```

Comment:
Not sure if this is right. No validation about the plugin instance. Maybe foreaching and using addPlugin?